pkg/etc/config.go

```diff
@@ -40,6 +51,28 @@ func GetAPIConfig() (cfg APIConfig, err error) {
 	return
 }
 
+func GetTLSConfig() (cfg TLSConfig, err error) {
+	err = env.Parse(&cfg)
+
+	cfg.RootCAs, _ = x509.SystemCertPool()
```

Comment:
Suggest explicitly handling any errors returned by functions (log it for troubleshooting in future)

Reply:
Added err handling as well as log warning in case root CAs cannot be loaded.